<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lbl_ftp" >
       <property name="text" >
        <string>ftp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ledit_ftp" >
       <property name="text" >
        <string>user@localhost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_pwd" >
       <property name="text" >
        <string>pwd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ledit_pwd" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lbl_http" >
       <property name="text" >
        <string>http</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ledit_http" >
       <property name="text" >
        <string>http://www.qualcosa.com/</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
